spring-integration-jdbc/src/main/sql/schema.sql.vpp

```diff
@@ -52,3 +52,11 @@ CREATE TABLE INT_CHANNEL_MESSAGE (
 
 CREATE INDEX INT_CHANNEL_MSG_DATE_IDX ON INT_CHANNEL_MESSAGE (CREATED_DATE, MESSAGE_SEQUENCE);
 CREATE INDEX INT_CHANNEL_MSG_PRIORITY_IDX ON INT_CHANNEL_MESSAGE (MESSAGE_PRIORITY DESC, CREATED_DATE, MESSAGE_SEQUENCE);
+
+
+CREATE TABLE INT_METADATA_STORE  (
+	METADATA_KEY ${VARCHAR}(255),
+	METADATA_VALUE ${VARCHAR}(4000),
+	REGION ${VARCHAR}(100) NOT NULL,
+	constraint METADATA_STORE primary key (METADATA_KEY)
```

Comment:
Not sure how my comment has been lost, but I'll do that one more time.

After introducing `REGION` we really should have composite PK, because different clients may use the same `METADATA_KEY`. See other tables definitions.
Also `METADATA_KEY` must not be `NULL` as well.